ext/GeometryOpsLibGEOSExt/GeometryOpsLibGEOSExt.jl

```diff
@@ -3,7 +3,7 @@ module GeometryOpsLibGEOSExt
 import GeometryOps as GO, LibGEOS as LG
 import GeoInterface as GI
 
-import GeometryOps: GEOS, enforce
+import GeometryOps: GEOS, enforce, _True, _False, _booltype
```

Comment:
Feels like we should remove the underscores if we are importing these?

Reply:
Good point.  We can probably make these uppercase, but only exported from Core (not GeometryOps proper)?

Reply:
Yep perfect

Reply:
Just realised we can use all of these in Rasters.jl too as its all the same there already. So no underscores is good.

Really keen to unify GeometryOps/Rasters around this core for all geometry related things now.